src/main/include/subsystems/ClimberHAL.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <rev/CANSparkMax.h>
+#include <subsystems/SubSystemConfig.h>
+#include <rev/SparkMaxAlternateEncoder.h>
+
+class ClimberHAL{
+    public:
+        ClimberHAL();
+        void RunClimber(double controllerSpeed);
+        bool isNextRotationTooHigh(double height);
+    private:
+        rev::CANSparkMax m_climberMotorLeft = rev::CANSparkMax(CLIMBER_MOTOR_LEFT, rev::CANSparkLowLevel::MotorType::kBrushless); 
+        rev::CANSparkMax m_climberMotorRight = rev::CANSparkMax(CLIMBER_MOTOR_RIGHT, rev::CANSparkLowLevel::MotorType::kBrushless); 
+
+        rev::SparkRelativeEncoder m_EncoderRight = m_climberMotorRight.GetEncoder(rev::SparkRelativeEncoder::Type::kHallSensor, 1);
+        rev::SparkRelativeEncoder m_EncoderLeft = m_climberMotorLeft.GetEncoder(rev::SparkRelativeEncoder::Type::kHallSensor, 1);
+    
+        rev::REVLibError error;
+
+        //Constants
+        const double wheel_radius = 1.5;
+        const double wheel_circumference = 2 * 3.14 * wheel_radius;
+        const double maxHeight = (3 + 8/12) * 12; //inches     
```

Comment:
const variables should be all caps... see subsystem config for example

Reply:
Done